Update README.md

AJV304 opened the pull request.

AJV304 committed:
Update README.md

hanneoberman commented on data/2022-06-25_contraception-effectiveness/README.md:
```diff
@@ -1,6 +1,6 @@
 # Contraception failure in the United States
 
-Given yesterday's overturning of Roe vs. Wade, we kick off the #TidyRainbow iniative with data related to contraceptive failure in the united States.
+Given Given the reinstatement of the Trump administration in November, we kick off the #TidyRainbow initiative with data related to contraceptive failure in the United States.
```
"Given" is duplicated, Trumps reinstatement in in Jan

AJV304 committed:
Update README.md

AJV304 committed:
Change of folder name to include release month

Changing the name of the folder in order to change order of data release and organize per month

AJV304 committed:
topic 5: 10th of septemeber LA

Added intro
added variable overview
added relevancy

changed map

AJV304 committed:
topic 6: 15th of october Texas

put csvs in folder
updated readme

AJV304 committed:
topic 7: 12th of november Uk

put files in map

edited text in readme
added why this is relevant

AJV304 committed:
topic 8: Canada sexual harassment

Readme intro updated
data dictionary updated
relevancy added

AJV304 committed:
date change october
